client/src/components/Layout/AppNavbar.vue

```diff
@@ -38,22 +104,59 @@ nav {
   background-color: v.$main-bkgrnd !important;
   height: 70px !important;
 
+  .v-btn {
+    padding: 0 !important;
+    text-transform: none;
+    letter-spacing: 0;
+    vertical-align: initial;
+
+    .mdi-chevron-down::before {
+      color: v.$main-white !important;
+    }
+  }
+
   .v-toolbar__content {
     height: 70px !important;
     justify-content: space-between;
   }
   &__logo {
     height: 40px;
   }
+
+  &__links {
+    display: inline-flex;
+    flex-wrap: nowrap;
+    align-items: center;
+    justify-content: space-between;
+    width: 270px;
+  }
```

Comment:
I would also like to space the links out enough so that the dropdowns all align to the left of the word like they do on Techtonic's site (currently About does not do this)